src/styles/index.scss

```diff
@@ -1,3 +1,147 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+h3,
+p {
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.card {
+  max-width: 200px;
+  width: 100%;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  &__img {
+    width: 100%;
+    margin-bottom: 40px;
+  }
+  &__prod {
+    margin-bottom: 16px;
+  }
+  &__rewie {
+    margin-bottom: 24px;
+  }
+  &__price {
+    margin-bottom: 16px;
+  }
+}
+
+.product {
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 150%;
+    color: #060b35;
+    margin-bottom: 4px;
+  }
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 140%;
+    color: #616070;
+  }
+}
+
+.rewiews {
+  display: flex;
+  justify-content: space-between;
+  height: 16px;
+  min-width: 116px;
+  width: 100%;
+  gap: 17px;
+
+  &__stars {
+    flex-grow: 1;
+  }
+
+  &__count {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 140%;
+    text-align: right;
+    color: #060b35;
+  }
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &--1 &__star:nth-child(1),
+  &--2 &__star:nth-child(-n + 2),
+  &--3 &__star:nth-child(-n + 3),
+  &--4 &__star:nth-child(-n + 4),
+  &--5 &__star:nth-child(-n + 5) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+
+  &__titel {
```

Comment:
There is a typo in the class name '&__titel'. It should be '&__title' to match the corrected class name in the HTML file.